src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -30,25 +30,28 @@ public Task<TContext> Run(TimeSpan? testExecutionTimeout = null)
             var scenarioContext = new TContext();
             contextInitializer(scenarioContext);
 
-            var runDescriptor = new RunDescriptor
-            {
-                ScenarioContext = scenarioContext
-            };
-            runDescriptor.Settings.Merge(settings);
```

Comment:
Give that this method is public, by removing the `Merge` and passing `RunSettings` in directly, could you run into a problem with sharing settings that wouldn't have been possible before? For example creating a single instance of `RunSettings` and then modifying between Run calls?